src/cryptoadvance/specter/templates/includes/dnd-textarea.html

```diff
@@ -0,0 +1,153 @@
+<!-- 
+    A textarea with drag and drop functionality.
+    Upon dragging and dropping the file in the textarea, the value of the textarea will be replaced by the text content stored in the file.
+    
+    Note: The default text within the textarea should be passed using 'value' attribute instead of enclosing the text within the opening and closing tag.
+
+{#- Usage -
+
+    <dnd-textarea
+        name = "name"
+        value = "some default text"
+        placeholder = "placeholder text"
+        textarea-style = "style to override default styling of textarea"
+    >
+    </dnd-textarea>
+
+#} 
+-->
+
+<template id="dnd-textarea">
+
+    <link rel="stylesheet" type="text/css" href="{{ url_for('static', filename='styles.css') }}">
+    <style>
+        :host {
+            display: block;
+            width: 100%;
+        }
+        #textarea-content { 
+            height: 150px; 
+            font-size: 13px;
+        }
+    </style>
+
+    <textarea
+        id="textarea-content"
+        placeholder="{{ _('You can paste and drag & drop here:') }}"
+    ></textarea>
+</template>
+
+<script type="text/javascript">
+    class DndTextareaElement extends HTMLElement {
+        static formAssociated = true;
+
+        constructor() {
+            super();    
+            this.internals = this.attachInternals();
+
+            // Create a shadow root
+            let shadow = this.attachShadow({mode: 'open'});
+            let template = document.getElementById('dnd-textarea');
+            let clone = template.content.cloneNode(true);
+
+            this.$textarea = clone.querySelector("textarea");
```

Comment:
Same question regarding the $ sign as here: https://github.com/cryptoadvance/specter-desktop/pull/1752#discussion_r897720045

Reply:
I added $ as a custom convention to differentiate between variables containing dom elements and variables containing properties, values and properties. Should I remove them?

Reply:
oh sure then, will do it